reference/7.2/Microsoft.PowerShell.Core/About/about_Pipeline_Chain_Operators.md

```diff
@@ -176,17 +176,17 @@ As of PowerShell 7, the behaviour of these syntaxes has been changed
 so that `$?` is set as expected when a command succeeds or fails
 within parentheses or a subexpression.
 
-Like most other operators in PowerShell, `&&` and `||` are also *left-associative*,
-meaning they group from the left. For example:
+Like most other operators in PowerShell, `&&` and `||` are also
+_left-associative_, meaning they group from the left. For example:
 
 ```powershell
 Get-ChildItem -Path ./file.txt || Write-Error "file.txt does not exist" && Get-Content -Raw ./file.txt
```

Comment:
I didn't make the change, but I considered using newlines here. Open to either format.
```suggestion
Get-ChildItem -Path ./file.txt ||
    Write-Error "file.txt does not exist" &&
    Get-Content -Raw ./file.txt
```